### ISSUE TITLE
Add reporting

### ISSUE THREAD

**Issue by lfolco** [kalenjordan/commercehero-issues#70]
Would love to have a way to export number of hours and total paid for each hero. Accountants like that kind of stuff.

**Comment by kalenjordan** [comment]
👍

**Comment by kalenjordan** [comment]
I was thinking more about this and I feel like this would probably fit better into an accounting software like xero or quickbooks? That's usually the best place to export reports for accountants, finance teams, etc.?

**Comment by lfolco** [comment]
True, but if you don't use those then you're left with doing it manually. I do a full invoice for my clients that includes not just the hero's work, so there's no way to break it out manually. What if I threw in a package of stroopwafels?

**Comment by kalenjordan** [comment]
I think most people tend to have some kind of system they use to track all their outgoing expenses like Xero or Quickbooks.

Stroopwafels???!!!  *starts coding immediately*

**Comment by lfolco** [comment]
Crap does that mean I need to grow up and start using accounting software?!?

**Comment by kalenjordan** [comment]
Time....she comes for us all.

**Comment by lfolco** [comment]
Turns out it's a manual process in QuickBooks as well (at least with Harvest integration).
